tests/Functional/DomainCommandsTest.php

```diff
@@ -43,16 +43,16 @@ public function testDomainAddListLookupRemove()
         $domains = array_column($domainList, 'id');
         $this->assertContains($testDomain, $domains, 'Domain list should contain added domain');
 
-        $lookUpResult = $this->terminusJsonResponse(sprintf('domain:lookup %s', $testDomain));
-        $this->assertIsArray($lookUpResult);
-        $this->assertNotEmpty($lookUpResult);
-        $this->assertArrayHasKey('site_id', $lookUpResult);
-        $this->assertNotEmpty($lookUpResult['site_id']);
-        $this->assertArrayHasKey('site_name', $lookUpResult);
-        $this->assertNotEmpty($lookUpResult['site_name']);
-        $this->assertEquals($this->getSiteName(), $lookUpResult['site_name']);
-        $this->assertArrayHasKey('env_id', $lookUpResult);
-        $this->assertEquals('test', $lookUpResult['env_id']);
+//        $lookUpResult = $this->terminusJsonResponse(sprintf('domain:lookup %s', $testDomain));
```

Comment:
Why are these tests commented out? Was this removed before being fixed in the other PR, perhaps?

Reply:
> Why are these tests commented out? Was this removed before being fixed in the other PR, perhaps?

Left commented by mistake (during debugging). Fixed.